exodus/exodus/settings/base.py

```diff
@@ -101,9 +100,9 @@
 EX_APK_FS_ROOT = os.path.join(EX_FS_ROOT, "apks")
 EX_NET_FS_ROOT = os.path.join(EX_FS_ROOT, "net")
 # Celery
-CELERY_RESULT_BACKEND = 'pyamqp://'
+CELERY_RESULT_BACKEND = 'rpc://'
 CELERY_ACCEPT_CONTENT = ['json']
-CELERY_TASK_SERIALIZER = 'json'
+CELERY_TASK_SERIALIZER = 'pickle'
```

Comment:
It seem `pickle` is "partially" deprecated, can we use `json` instead ?
Source: https://docs.celeryproject.org/en/latest/userguide/security.html#serializers

Reply:
I had to change otherwise I was getting errors about JSON serialization when creating new analysis. If there is any other way to fix that error, I'm happy to change it :)